### DISCUSSION TITLE
函数如何返回两列输出

### DISCUSSION THREAD

**Discussion by TaoZex** [apache/doris#12680] [A - General / Q&A]
最近需要实现一个函数添加到社区。该函数的功能如下:

<img width="576" alt="1" src="https://user-images.githubusercontent.com/45089228/190629068-2e653b92-5282-4847-8ac8-e8cd8ff93a2a.png">
对于每一行数据，将生成两列输出，第一列为下标(从0开始递增)，第二列带有explode_split函数的结果。

我在Doris社区看到了explode_split的源代码:
https://github.com/apache/doris/blob/master/be/src/exprs/table_function/explode_split.cpp
并咨询了源码的作者，目前doris好像并不支持返回两列，想咨询一下有没有其他办法实现返回两列输出呢？

**Comment by smallhibiscus** [comment]
Hi,Abount the poseexplode function, I would like to talk to you with detail. Could you add my WeChat No, my WeChat id `smallhibiscus`.
